lib/timeline/component/timeline-chart/TimelineChartDataAxis.js

```diff
@@ -21,7 +21,7 @@ class TimelineChartDataAxis extends DataAxis {
       labelOffsetX: 10,
       labelOffsetY: 2,
       iconWidth: 20,
-      width: parseInt(document.querySelector('.data-region.data-container-with-timeline').offsetWidth / 26) + 'px',
+      width: (document.querySelector('.data-region.data-container-with-timeline')) ? parseInt(document.querySelector('.data-region.data-container-with-timeline').offsetWidth / 26) + 'px' : '0px',

```

Comment:
Please not execute twice document.querySelector('.data-region.data-container-with-timeline')